### PR TITLE
add option to specify cmd health check

### PR DESCRIPTION
**Pull request by miltontony**
mission_control_health_cmd

### CODE REVIEW

**Comment by miltontony** [comment]
@bearnard @saeedmb8 review please

File: mc2/controllers/docker/forms.py [hunk 1]
```diff
@@ -45,6 +50,18 @@ class DockerControllerForm(ControllerForm):
...
+    def clean(self):
+        cleaned_data = super(DockerControllerForm, self).clean()
+        marathon_health_check_cmd = cleaned_data.get(
+            "marathon_health_check_cmd")
+        marathon_health_check_path = cleaned_data.get(
+            "marathon_health_check_path")
+
+        if marathon_health_check_path and marathon_health_check_cmd:
+            raise forms.ValidationError(
+                "You can only specify 1 health check at a time."
```
Comment (bearnard) [inline mc2/controllers/docker/forms.py]: I that something we enforce or the marathon api?

File: mc2/controllers/docker/models.py [hunk 2]
```diff
@@ -112,6 +114,23 @@ def get_marathon_app_data(self):
                         int(settings.MESOS_DEFAULT_TIMEOUT_SECONDS),
                 }]
             })
+        elif self.marathon_health_check_cmd:
+            app_data.update({
+                "ports": [0],
```
Comment (bearnard) [inline mc2/controllers/docker/models.py]: Why are we setting ports here?

File: mc2/controllers/docker/forms.py (hunk 1 above)
Comment (miltontony) [reply mc2/controllers/docker/forms.py]: It's actually just us.. I'll remove this limitation actually..

File: mc2/controllers/docker/models.py (hunk 2 above)
Comment (miltontony) [reply mc2/controllers/docker/models.py]: 🤔 artefact from other health check -- I'll remove too

**Comment by bearnard** [comment]
👍 when done with the changes you suggested.

**Pull request merged**